src/util/sandbox/fd_sandbox.c

```diff
@@ -0,0 +1,256 @@
+#ifdef FD_HAS_SANDBOX
+
+#ifndef _GNU_SOURCE
+#define _GNU_SOURCE
+#endif
+
+#include "fd_sandbox.h"
+#include "fd_sandbox_private.h"
+
+#include <errno.h>        /* errno */
+#include <fcntl.h>        /* open */
+#include <linux/audit.h>
+#include <linux/filter.h>
+#include <linux/seccomp.h>
+#include <sched.h>        /* CLONE_*, setns, unshare */
+#include <stddef.h>
+#include <stdio.h>        /* snprintf */
+#include <stdlib.h>       /* clearenv, mkdtemp*/
+#include <sys/mount.h>    /* MS_*, MNT_*, mount, umount2 */
+#include <sys/prctl.h>
+#include <sys/resource.h> /* RLIMIT_*, rlimit, setrlimit */
+#include <sys/stat.h>     /* mkdir */
+#include <sys/syscall.h>  /* SYS_* */
+#include <unistd.h>       /* set*id, sysconf, close, chdir, rmdir syscall */
```

Comment:
Willing to bet there is a bunch of cleanups and compression that can be done here.  (Also, the linux/filter.h is redundant with previous, but this is actually the place where it actually should be.)